front/lib/models.ts

```diff
@@ -929,10 +928,6 @@ GensTemplate.init(
       allowNull: false,
       defaultValue: DataTypes.NOW,
     },
-    instructions: {
```

Comment:
please double check the column do get removed when connecting directly to the DB?